test/unit/middlewares/idValidation.spec.js

```diff
@@ -0,0 +1,17 @@
+const idValidation = require('~/middlewares/idValidation')
+const { createError } = require('~/utils/errorsHelper')
+const { INVALID_ID } = require('~/consts/errors')
+
+describe('Validation middleware', () => {
```

Comment:
I think it's better to name it `ID validation middleware` to not be confused with the next test suit.

Reply:
fixed